app/assets/javascripts/application/submission.js

```diff
@@ -4,49 +4,77 @@ var mumuki = mumuki || {};
   function ResultsBox(submissionsResults) {
     this.submissionsresultsArea = submissionsResults;
     this.processingTemplate = $('#processing-template');
+    this.submissionsErrorTemplate = $(".submission-result-error");
   }
 
   ResultsBox.prototype = {
     waiting: function () {
-      console.log('submitting solution');
       this.submissionsresultsArea.html(this.processingTemplate.html());
+      this.submissionsErrorTemplate.hide();
     },
     success: function (data) {
       this.submissionsresultsArea.html(data);
     },
-    error: function (error) {
-      this.submissionsresultsArea.html('<pre>' + error + '</pre>');
+    error: function () {
+      this.submissionsresultsArea.html('');
```

Comment:
It should be `submissionsResultsArea`

Reply:
Use el nombre que estaba, ahora lo renombro